### PR TITLE
Improve error message and fix ns for manager-internal-tls

### PR DESCRIPTION
**Pull request by rene-dekker**
Two improvements:
- Manager internal secret is now created and read from the operator namespace, rather than the calico namespace.
- Error messages regarding misssing PEM and  missing DNS names now include the namespace and name of the secret that is causing the error.

### CODE REVIEW

**Review by Brian-McM — APPROVED** [review]

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Comment by marvin-tigera** [comment]
I'm sorry but I failed delete the branch after merging the pull request.

**Pull request merged**
